### PR TITLE
DOC: Some updates to the array_api compat document

### PR DESCRIPTION
**Pull request by asmeurer**

### CODE REVIEW

File: doc/source/reference/array_api.rst [hunk 1]
```diff
@@ -153,7 +153,9 @@ Keyword Argument Renames
 The following functions have keyword arguments that have been renamed. The
 functionality of the keyword argument is identical unless otherwise stated.
 Each new keyword argument is not already present on the given function in
```
Comment (charris) [inline doc/source/reference/array_api.rst:155]: This is difficult to parse, could you rephrase it?

Comment (asmeurer) [reply doc/source/reference/array_api.rst:155]: I think I meant to delete this sentence in favor of the one I added.

**Comment by charris** [comment]
Thanks Aaron.

**Comment by agoose77** [comment]
Hope this isn't creating too much noise — I noticed that `reshape` [takes a `copy` argument](https://data-apis.org/array-api/2021.12/API_specification/generated/array_api.reshape.html) that [we don't replicate](https://github.com/numpy/numpy/blob/77afec98627952c5bc99dc6e394069de73246c6c/numpy/array_api/_manipulation_functions.py#L56) in NumPy? Is this a deliberate difference? Thanks for the work here all!

**Comment by asmeurer** [comment]
@agoose77 I think that is something that just fell through the cracks. Can you open an issue (or PR) about it?

**Comment by agoose77** [comment]
Sure!
